### PR TITLE
Added models tables, validations and associations

### PR DESCRIPTION
**Pull request by onofrecc**
We've just added the bike model and booking model, updated the user model, created associations and validations, updated the schema

### CODE REVIEW

**Comment by A-drienne** [comment]
It's amazing!!!!

**Comment by alvarotrujillo** [comment]
Looks great, good job!

File: db/migrate/20220822123157_create_bikes.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+class CreateBikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bikes do |t|
+      t.string :name
+      t.string :type
+      t.string :location
+      t.text :description
+      t.string :gears
+      t.boolean :hand_brakes
+      t.float :price
```
Comment (alvarotrujillo) [inline db/migrate/20220822123157_create_bikes.rb]: Maybe change the price to an integer

File: db/migrate/20220822123157_create_bikes.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+class CreateBikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :bikes do |t|
+      t.string :name
+      t.string :type
+      t.string :location
+      t.text :description
+      t.string :gears
+      t.boolean :hand_brakes
+      t.integer :price
```
Comment (A-drienne) [inline db/migrate/20220822123157_create_bikes.rb:10]: changed price to integer

**Review by alvarotrujillo — COMMENTED** [review]
Looks great, I would consider changing the price to an integer rather than float.

**Pull request merged**